src/protocol/prf_sharding/mod.rs

```diff
@@ -799,6 +826,40 @@ pub mod tests {
         }
     }
 
+    impl IntoShares<CappedAttributionOutputs> for PreAggregationTestOutput {
+        fn share_with<R: Rng>(self, rng: &mut R) -> [CappedAttributionOutputs; 3] {
```

Comment:
Why is this now required?

Reply:
Needed for do_aggregation and move_bucket test which splits BitDecomposed<Gf2> into its shares